test/e2e/common/node/pod_resize.go

```diff
@@ -894,8 +894,8 @@ func doPodResizeTests(f *framework.Framework) {
 
```

Comment:
Worth adding a test that shows using RBAC to only allow subject to update /resize, but not the entire pod, and then verify that the subject is able to update via resize but not by writes to the root pod reosurce?

Reply:
Done. But I added integration tests instead of e2e tests.